src/main/java/org/soen343/controllers/SmartHomeCoreModuleController.java

```diff
@@ -167,4 +170,25 @@ public void ToggleAutoMode(ActionEvent actionEvent) {
         autoModeButton.setText(status ? "Auto Mode ON" : "Auto Mode OFF");
         autoModeButton.setSelected(status);
     }
+
+    public void disableButtons(){
+        autoModeButton.setDisable(true);
+        doorsTreeView.setDisable(true);
+        windowsTreeView.setDisable(true);
+        lightsTreeView.setDisable(true);
+        doorsButton.setDisable(true);
+        lightsButton.setDisable(true);
+        windowsButton.setDisable(true);
+    }
+
+    public void enableButtons(){
+        autoModeButton.setDisable(false);
+        doorsTreeView.setDisable(false);
+        windowsTreeView.setDisable(false);
+        lightsTreeView.setDisable(false);
+        doorsButton.setDisable(false);
+        lightsButton.setDisable(false);
+        windowsButton.setDisable(false);
+    }
```

Comment:
Instead of disabling the individual button, can we disable the whole pane, by disabling the anchor pane.

Reply:
yeah but I didn't disable the tables, and the user should still be able to change panes even if the simulation is on/off

Reply:
Disabling the anchor pane will not block the user to change pane